cmd/dptp-controller-manager/main.go

```diff
@@ -271,6 +274,10 @@ func main() {
 
 	var errs []error
 	for cluster, cfg := range kubeconfigs {
+		// no jobs run on the hive cluster and hive is not considered a build cluster
```

Comment:
This smells. If we have a controller that operates on the state of one cluster, it should not be in the core manager and then `if`-checked out of running the normal way. We should just deploy a the controller we want on `hive`.

Reply:
(that furthermore allows us to to simplify the deployment significantly by e.g. using in-cluster config and SA tokens without involving the entire cluster->vault->cluster sync)

Reply:
I second this: it looks like we'll eventually write several other pool-related controllers, so having a separate binary just for that purpose (`dptp-pools-cm`?) sounds like a good idea.

Reply:
I will do a new cli: `dptp-pools-cm`.